Makefile

```
@@ -0,0 +1,36 @@
+FLASK_PORT=5000
+PID_FILE=flask_server.pid
+SWAGGER_FILE=swagger.yaml
+
+# Run the Flask server
+run:
+	@echo "Starting Flask server..."
+	@FLASK_APP=$(FLASK_APP) FLASK_ENV=$(FLASK_ENV) flask run --host=0.0.0.0 --port=$(FLASK_PORT) & echo $$! > $(PID_FILE)
+	@echo "Server is running at http://127.0.0.1:$(FLASK_PORT)"
+
+# Stop the Flask server
+stop:
```

checkmake (0.2.2)

[warning] Target body for "stop" exceeds allowed length of 5 (7).

(maxbodylength)